README.md

```diff
@@ -1,24 +1,20 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+# Koroibos Olympian Analytics Platform
+
+## Details
+
+To be written...
```

Comment:
This will be expanded upon as time goes on, mostly just a little blurb about the scope of the project, learning goals, Turing, etc...